core/src/main/java/com/novoda/noplayer/exoplayer/mediasource/ExoPlayerTrackSelector.java

```diff
@@ -22,55 +15,25 @@ public ExoPlayerTrackSelector(DefaultTrackSelector trackSelector) {
         this.trackSelector = trackSelector;
     }
 
-    public void selectAudioTrack(PlayerAudioTrack audioTrack) {
-        MappingTrackSelector.MappedTrackInfo trackInfo = trackSelector.getCurrentMappedTrackInfo();
-        TrackGroupArray trackGroups = trackInfo.getTrackGroups(TRACK_TYPE_AUDIO);
-        FixedTrackSelection.Factory factory = new FixedTrackSelection.Factory();
-
-        MappingTrackSelector.SelectionOverride selectionOverride = new MappingTrackSelector.SelectionOverride(
-                factory,
-                audioTrack.groupIndex(),
-                audioTrack.formatIndex()
-        );
-        trackSelector.setSelectionOverride(TRACK_TYPE_AUDIO, trackGroups, selectionOverride);
+    TrackGroupArray getAudioTrackGroups() {
```

Comment:
This class is just a wrapper around methods that cannot be mocked because the underlying data type is final.